### PR TITLE
[monorepo] chore: Remove usages of Magmo in files

### LINKED ISSUE #533: [devtools] Package.json out of date - Refers to Magmo
The package.json for packages/devtools still points to Magmo repo.
This is the file:
https://github.com/statechannels/monorepo/blob/master/packages/devtools/package.json

### PR DESCRIPTION
**Pull request by Alonski**
### Description

Lots of ♥ for Magmo but links were pointing to incorrect places 😅

This fixes usages of Magmo in files such as package.json and READMEs.

There are still a few links pointing to places that don't have a corresponding file in StateChannels

### Related issues

Closes #533

### CODE REVIEW

File: packages/devtools/README.md [hunk 1]
```diff
@@ -7,13 +7,13 @@ Developer tooling, used by statechannels packages.
...
 ```
-$ npm i -D https://github.com/magmo/devtools
+$ npm i -D https://github.com/statechannels/monorepo/blob/master/packages/devtools
 ```
 
 or
 
 ```
-$ yarn add -D https://github.com/magmo/devtools
+$ yarn add -D https://github.com/statechannels/monorepo/blob/master/packages/devtools
```
Comment (geoknee) [inline packages/devtools/README.md]: Do these commands actually work?

Comment (Alonski) [reply packages/devtools/README.md]: I am not sure... I didn't actually test them :)

Should we even teach how to install this as its internal?

File: packages/jest-gas-reporter/package.json [hunk 2]
```diff
@@ -4,7 +4,10 @@
   "description": "A jest reporter that reports the gas used by various calls to ethereum contracts.",
   "main": "lib/src/index.js",
   "types": "lib/src/index.d.ts",
-  "repository": "https://github.com/magmo/jest-gas-reporter.git",
+  "repository": {
+    "type": "git",
+    "url": "http://github.com/statechannels/monorepo/blob/master/packages/jest-gas-reporter"
```
Comment (tomclose) [inline packages/jest-gas-reporter/package.json]: Why the `blob/master` here?

File: packages/devtools/README.md (hunk 1 above)
Comment (geoknee) [reply packages/devtools/README.md]: I tried them: they don't work. I would just remove this section for now.

**Comment by Alonski** [comment]
Thanks @geoknee !

**Pull request merged**